storage/pruning/pruningStorer.go

```diff
@@ -245,11 +245,16 @@ func (ps *PruningStorer) Put(key, data []byte) error {
 			log.Debug("active persister not found",
 				"epoch", ps.epochForPutOperation,
 				"used", persisterToUse.epoch)
+
+			// Question for review: isn't this a critical error?
```

Comment:
I think it is critical because otherwise you'd save data in a persister which is not for the desired epoch

Reply:
If we return error here, the node won't advance: this will keep the DB as consistent as possible. If we not, the node will advance, but the data might not be stored inside

Reply:
Understood. Left as it were - to prevent breaking something unforeseen. However, let me know if I should change instead.

Reply:
this case didn't happen because the `epochForPutOperation` is set on `CommitBlock` so there are very small chances that the persister for the specified epoch doesn't exist. can remain like this for now